lib/hammer_cli_katello/repository.rb

```diff
@@ -11,7 +11,7 @@ class ListCommand < HammerCLIKatello::ListCommand
           field :name, _("Product")
         end
         field :content_type, _("Content Type")
-        field :url, _("URL")
+        field :relative_path, _("URL")
```

Comment:
Oops, lost some code there. I will update to include the relative path only in the 'hammer repository info' rather than in the list.